resdk/tests/unit/test_resolwe.py

```diff
@@ -443,7 +450,8 @@ def setUp(self, auth_mock):  # pylint: disable=arguments-differ
     def test_bad_url(self, requests_mock):
         requests_mock.post = MagicMock(side_effect=[requests.exceptions.ConnectionError()])
 
-        with six.assertRaisesRegex(self, ValueError, r"Server not accessible on .*. Wrong url?"):
+        with six.assertRaisesRegex(self, ValueError,
+                                   'Server not accessible on www.abc.com. Wrong url?'):
```

Comment:
This is how you check literals as well. Look at the example of [Python's assertRaisesRegexp here](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaisesRegexp). We use the implementation in six to be compatible with py2 and py3.